perf/aiostress.py.data/aio.yaml

```diff
@@ -0,0 +1 @@
+url : 'https://github.com/linux-test-project/ltp/blob/master/testcases/kernel/io/ltp-aiodio/aio-stress.c'
```

Comment:
@PraveenPenguin can we change the yaml file name matching with .py file.
it might help in upcoming automation and will be in sync with test names?

Reply:
sure Naresh will do